src/fiskaltrust.Middleware.Demo/Program.cs

```diff
@@ -23,7 +23,7 @@ public static class Program
         /// <param name="cashboxId">The cashboxid for the Middleware.</param>
         /// <param name="url">The url that is used for sending requests to the Middleware(Default: grpc://localhost:10103).</param>
         /// <returns></returns>
-        public static async Task Main(string cashboxId, string url = "grpc://localhost:10103")
+        public static async Task Main(string cashboxId= "54c6b434-cd27-442e-b39f-0960c4ad1bda", string url = "rest://localhost:1500/4417bfc4-cb55-4813-8914-1665cff5187e")
```

Comment:
I think this was for testing purposes :) maybe we should revert this to the old settings